extensions/chat/common/client.mjs

```diff
@@ -14,7 +14,7 @@ class ChatOverlay extends Overlay {
 
   // TODO(zeze-zeze): Expand on this class (jsdoc and implementation) and
   // implement onPre/PostShow/Hide() as needed.
-};
+}
```

Comment:
Is this correct?

Reply:
sorry for none description 
all fixed by eslint --fix